tests/functions/test_base.py

```diff
@@ -1,10 +1,29 @@
+import os
+
+import mock
 import pytest
 
 from exodus_lambda.functions.base import LambdaBase
 
-CONF_PATH = "exodus_lambda/functions/lambda_config.json"
+CONF_PATH = "configuration/lambda_config.json"
 
 
-def test_base():
+def test_base_handler():
     with pytest.raises(NotImplementedError):
         LambdaBase(conf_file=CONF_PATH).handler(event=None, context=None)
+
+
+@pytest.mark.parametrize(
+    "region",
+    ["us-south-7", "us-east-1", "ap-northeast-3"],
+    ids=["fake", "available", "unavailable"],
+)
+def test_base_region(region):
+    """Ensure correct regions are selected for various inputs"""
+    with mock.patch.dict(os.environ, {"AWS_REGION": region}):
+        base = LambdaBase(conf_file=CONF_PATH)
+        # Ensure region defaults when environment variable is set
+        assert base.region == "us-east-1"
```

Comment:
Because us-east-1 is the default region, it seems like there is no test which verifies that the env var actually does anything.  This is the only test being added, and it seems like if you delete the code which reads the env var or change it to read from the wrong env var, then this test would still pass... or did I miss something?

Could we have at least one test which verifies that the value from AWS_REGION is actually used where possible?